addon/mixins/authenticated-route-mixin.js

```
@@ -57,7 +57,11 @@ export default Ember.Mixin.create({
       transition.abort();
       this.get('session').set('attemptedTransition', transition);
       Ember.assert('The route configured as Configuration.authenticationRoute cannot implement the AuthenticatedRouteMixin mixin as that leads to an infinite transitioning loop!', this.get('routeName') !== Configuration.authenticationRoute);
-      this.transitionTo(Configuration.authenticationRoute);
+      try {
+        this.transitionTo(Configuration.authenticationRoute);
+      } catch(e) {
+        Ember.Logger.error(e.message);
```

This should re-throw the error.